### PR TITLE
✔ Fix the App Folders Structure

### PR DESCRIPTION
**Pull request by farhatamiine**

### CODE REVIEW

File: .idea/codeStyles/codeStyleConfig.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (adilbim) [inline .idea/codeStyles/codeStyleConfig.xml:1]: what is this file ??

File: .idea/color-app.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (adilbim) [inline .idea/color-app.iml:1]: and this ?

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (adilbim) [inline .idea/vcs.xml:1]: i didn't get the reason of this `idea` folder!

**Review by adilbim — CHANGES_REQUESTED** [review]

**Comment by adilbim** [comment]
why there is a white space around the app ? i think u deleted something in the css files.

image

File: .idea/vcs.xml (hunk 3 above)
Comment (farhatamiine) [reply .idea/vcs.xml:1]: something used inside the webstorm ide which u don't know cuz ur using vscode even if he has .vscode file

**Pull request merged**